sample_tracking/app/import-samples.tsx

```diff
@@ -203,11 +210,15 @@ export default function ImportSamples() {
                 });
                 const codeList = {};
                 let foundErrors = false;
+                let errorsArray:any = [];
+                let row = 0
                 results.data.forEach((result) => {
                     const errors = validateImportedEntry(result as Sample, errorMessages);
+                    console.log("Errors found while validating imported entry" + errors)
                     if (errors.length > 0) {
                         result.errors = errors;
                         foundErrors = true;
+                        errorsArray.push({row: row, error: errors})
```

Comment:
there was also a feature that Josh did where we would re-export the improperly formatted file with the errors added as a new column.  Is that still working?

Reply:
yes it is still working. The error bar will provide a link to download the csv with the errors.